File: U1/TI1.4 RESTFULL_DOTNET_SINBDD_GR01/02CLIWEB/ConUni_CLIWEB_Rest/ec/edu/monster/models/ErrorConversion.cs
using System;

namespace ConUni_CLIWEB_Rest.ec.edu.monster.models
{
    /// <summary>
    /// Modelo para errores de conversión
    /// </summary>
    public class ErrorConversion
    {
        public string CodigoError { get; set; }
        public string Mensaje { get; set; }
        public string TipoError { get; set; }
        public double? ValorProblematico { get; set; }
        public string Unidad { get; set; }
        public DateTime FechaError { get; set; }
        public string Detalles { get; set; }

        public ErrorConversion()
        {
            CodigoError = string.Empty;
            Mensaje = string.Empty;
            TipoError = string.Empty;
            Unidad = string.Empty;
            Detalles = string.Empty;
            FechaError = DateTime.Now;
        }
    }
}

File: U1/TI1.3 RESTFULL_JAVA_SINBDD_GR01/02 CLIWEB/ConUni_Rest_Web_GR01/src/main/java/ec/edu/monster/model/ErrorConversion.java
package ec.edu.monster.model;

/**
 * Modelo para errores de conversión
 * @author YourName
 */
public class ErrorConversion {
    private String mensaje;
    private String codigo;
    private String tipo;
    
    // Getters y Setters
    public String getMensaje() {
        return mensaje;
    }
    
    public void setMensaje(String mensaje) {
        this.mensaje = mensaje;
    }
    
    public String getCodigo() {
        return codigo;
    }
    
    public void setCodigo(String codigo) {
        this.codigo = codigo;
    }
    
    public String getTipo() {
        return tipo;
    }
    
    public void setTipo(String tipo) {
        this.tipo = tipo;
    }
}